database/sql/init.sql

```diff
@@ -134,27 +136,27 @@ VALUES
     (19, 19, 2, 'Zabezpieczanie systemów przed zagrożeniami cybernetycznymi', 250.00, 180.00, 'Elastyczne', 40.00, 'Stacjonarne'),
     (20, 20, 4, 'Rozwinięte rozwiązania e-commerce i sklepy internetowe', 200.00, 150.00, 'Poniedziałek-Piątek', 30.00, 'Mobilne');
 
-INSERT INTO Ratings (user_id, rating) VALUES
-    (1, 4),
-    (2, 2),
-    (3, 5),
-    (4, 1),
-    (5, 3),
-    (6, 5),
-    (7, 2),
-    (8, 4),
-    (9, 3),
-    (10, 1),
-    (11, 4),
-    (12, 2),
-    (13, 5),
-    (14, 1),
-    (15, 3),
-    (16, 5),
-    (17, 2),
-    (18, 4),
-    (19, 3),
-    (20, 1);
+INSERT INTO Ratings (specialist_id, reviewer_id, rating, comment) VALUES
```

Comment:
nie wiem czy nie warto wrócić do konceptu, gdzie to jest "user_id" a nie "specialist_id" bo powiedzieliśmy sobie, że nie rozróżniamy tego (IDki są te same dla userów i dla specjalistów) - specjalistą jesteś jeśli flaga "is_providing_services" jest True.

Reply:
oki, zmieniam to